src/main/java/kitchenpos/product/domain/ProductEntity.java

```diff
@@ -43,6 +43,11 @@ public BigDecimal getPrice() {
     return price.getValue();
   }
 
+  public BigDecimal calculateAmountWithQuantity(Long quantity) {
+    return price.getValue()
+        .multiply(BigDecimal.valueOf(quantity));
+  }
```

Comment:
getter 사용! 다음과 같이 개선해 볼 수 있을 것 같아요! 😄 
```suggestion
  public BigDecimal calculateAmountWithQuantity(Long quantity) {
    return price.multiply(quantity);
  }
```